subcommand/net-dial/command.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package netdial
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/mitchellh/cli"
+)
+
+type Command struct {
+	UI cli.Ui
+}
+
+func (c *Command) Run(args []string) int {
+	if len(args) != 2 {
+		c.UI.Error("invalid invocation, expected two positional args: <host> <port>")
+		return 1
+	}
+
+	conn, err := net.Dial("tcp", fmt.Sprintf("%s:%s", args[0], args[1]))
+	if err != nil {
+		return 1
+	}
+	conn.Close()
+
+	return 0
+}
+
+func (c *Command) Synopsis() string {
+	return "Performs a simple health check by opening a TCP connection to a host and port"
+}
+
+func (c *Command) Help() string {
+	return `usage: consul-ecs net-dial <host> <port>
```

Comment:
Completely agree with this. My initial thought was to keep it the same as the `nc` args, but this doesn't really make sense since it is not actually `nc`. I've changed it as suggested.